rules/config

```diff
@@ -226,10 +226,12 @@ SONIC_ENABLE_SECUREBOOT_SIGNATURE ?= n
 # SECURE_UPGRADE_DEV_SIGNING_CERT - path to development signing certificate, used for image signing during build
 # SECURE_UPGRADE_MODE - enum value for secure upgrade mode, valid options are "dev", "prod" and "no_sign"
 # SECURE_UPGRADE_PROD_SIGNING_TOOL - path to a vendor signing tool for production flow.
+# SECURE_UPGRADE_PROD_TOOL_ARGS - Extra arguments options for vendor to use to run with his specific prod signing tool
```

Comment:
Extra arguments options for vendor to use in order to run his specific prod signing tool